src/NServiceBus.AcceptanceTests/ScenarioDescriptors/AllTransports.cs

```diff
@@ -14,24 +14,7 @@ protected AllTransports()
             AddRange(ActiveTransports);
         }
 
-        static IEnumerable<RunDescriptor> ActiveTransports
-        {
-            get
-            {
-                if (activeTransports == null)
-                {
-                    //temporary fix until we can get rid of the "AllTransports" all together
-                    activeTransports = new List<RunDescriptor>
-                    {
-                        Transports.Default
-                    };
-                }
-
-                return activeTransports;
-            }
-        }
-
-        static ICollection<RunDescriptor> activeTransports;
+        static IEnumerable<RunDescriptor> ActiveTransports => new List<RunDescriptor> { Transports.Default };
```

Comment:
Is it required for this to allocate a new list every time? 
How about
`static IEnumerable<RunDescriptor> ActiveTransports { get; } = new List<RunDescriptor> { Transports.Default };`
instead?